delivery_admin/views/react/orders.jsx

```diff
@@ -19,48 +19,44 @@ export class Orders extends React.Component {
   }
 
   count_orders(orders){
-    var count = {}
+    let count = {}
     for(var i in orders){
-      var meal = orders[i].meal
+      let meal = orders[i].meal
       count[meal] = count[meal] ? count[meal]+1 : 1
     }
     return count
   }
 
   render () {
-
     let {orders} = this.state
     let count = this.count_orders(orders)
-
     return (
       <div>
         <h2>Orders per Meal Type</h2>
-
         <table className="table table-bordered">
-                  <thead>
-                    <tr>
-                      <th>Meal</th>
-                      <th>Amount</th>
-                    </tr>
-                  </thead>
-                  <tbody>
-                  {
-                    Object.keys(count).map((meal,i) => {
```

Comment:
I think you missed my previous comment on this. Use underscore js to do this. it's already an installed module (const _ = require('underscore') and you are good to go). With underscore you can obtain the same result by doing _.map(count, (value, key) => {...}) . if you want to know more check out the second example from here http://underscorejs.org/#map